src/main/java/org/medicmobile/webapp/mobile/SmsSender.java

```diff
@@ -107,6 +107,7 @@ private String describe(Intent intent) {
 		return intents;
 	}
 
+	@SuppressLint("UnspecifiedImmutableFlag")
```

Comment:
It is a flag that warns you about a bad practice of how we handle Pending intents, that are a kind of Promise of Intents in Android. Starting Android 12 we will need to change that, but it is not a requirement right now, so I took note to report the needed change in a separated ticket because fix it now may take time to develop and to test, delaying the rest of the changes that are needed ASAP.

Going to create the ticket now...

Reply:
got it, thanks

Reply:
Added the issue here: https://github.com/medic/cht-android/issues/205